drivers/bitsten.js

```diff
@@ -0,0 +1,26 @@
+const request = require('../lib/request');
+const Ticker = require('../models/ticker');
+const { parseToFloat } = require('../lib/utils');
+
+module.exports = async () => {
+  const { result: markets } = await request(
+    'https://api.bitsten.com/api/v1/public/getticker/all',
+  );
+
+  return Object.keys(markets).map((market) => {
+    const ticker = markets[market];
+    const [base, quote] = market.split('_');
+
+    return new Ticker({
+      base,
+      quote,
+      high: parseToFloat(ticker.high),
+      low: parseToFloat(ticker.low),
+      close: parseToFloat(ticker.last),
+      bid: parseToFloat(ticker.bid),
+      ask: parseToFloat(ticker.ask),
+      baseVolume: parseToFloat(ticker.basevolume),
```

Comment:
If I check their `ltc_btc` market they have the same value for `basevolume` as for `btcvolume`. Therefore I think the their base volume is actually the quote volume :smile:

Reply:
@nickpater it is the same for `ltc_btc`, but not for zrx_eth. That means that we do not have `baseVolume`, but `quoteVolume` we do (because ther `btcvolume` is our `quoteVolume`) ?

Reply:
Yes true! We can only use the `quoteVolume`, but thats fine!